package.json

```diff
@@ -18,20 +18,30 @@
   ],
   "nativescript": {
     "platforms": {
-      "ios": "1.5.0",
-      "android": "1.5.0"
+      "ios": "3.0.0",
+      "android": "3.0.0"
+    },
+    "tns-ios": {
+      "version": "3.0.0"
     }
   },
-  "author": "Alexander Vakrilov",
+  "author": {
+    "name": "Alexander Vakrilov"
+  },
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/vakrilov/native-script-accelerometer.git/issues"
   },
   "homepage": "https://github.com/vakrilov/native-script-accelerometer",
   "dependencies": {
-    "tns-core-modules": "^2.0.0"
+    "tns-core-modules": "^3.0.0"
+  },
+  "peerDependencies": {
+    "tns-core-modules": "^3.0.0"
```

Comment:
The code of the plugin is compatible with NativeScript 2.x. 
Lets make this dependency to `>2.0.0` instead of `^3.0.0` so that the plugin can still be used with older versions of NS